gulpfile.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+var gulp = require('gulp');
+var babel = require('gulp-babel');
+var concat = require('gulp-concat');
+var uglify = require('gulp-uglify');
+var size = require('gulp-size');
+var rename = require('gulp-rename');
+var mocha = require('gulp-mocha');
+var del = require('del');
+
+require('babel-core/register');
+
+var src = [
+  'src/shady-css/common.js',
+  'src/shady-css/node-factory.js',
+  'src/shady-css/node-visitor.js',
+  'src/shady-css/stringifier.js',
+  'src/shady-css/token.js',
+  'src/shady-css/tokenizer.js',
+  'src/shady-css/parser.js',
+  'src/*.js'
+];
+var dest = 'dist/';
+
+gulp.task('default', ['clean', 'test', 'build', 'minify', 'measure']);
+
+gulp.task('build', ['test'], function() {
+  return gulp.src(src)
+    .pipe(babel({
+      modules: 'umd',
+      loose: true,
+      blacklist: ['regenerator', 'es6.forOf']
```

Comment:
You upgraded to Babel 6, yes? I don't think Babel 6 support the blacklist or modules options. Are you getting errors?